### PR TITLE
config: Add browser parameter

### LINKED ISSUE #3701: config: Add a new `config.json` option to disable browser
Bring a new config option to disable browser through `config.json` , requires bumping of the config version to '14'. 

## Expected Behavior
```json
"browser": "disable",
....
```

In `config.json`

## Current Behavior
No way to disable minio browser without just ENV variable.

## Possible Solution
Implement support for `config.json` to dictate if browser should be turned off.

### PR DESCRIPTION
**Pull request by vadmeste**
## Description
browser new param receives "on" or "off" parameter which is similar
to MINIO_BROWSER

Parameter validation will be done in a separate PR which validates all params

## Motivation and Context
Fixes #3701 

## How Has This Been Tested?
go test

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: cmd/utils.go [hunk 1]
```diff
@@ -238,3 +241,25 @@ func dumpRequest(r *http.Request) string {
...
+}
+
+//
+func getBrowserFromEnv() (string, error) {
+	b := os.Getenv("MINIO_BROWSER")
+	if strings.TrimSpace(b) == "" {
+		return "", nil
+	}
+	if !strings.EqualFold(b, "off") && !strings.EqualFold(b, "on") {
+		return "", errors.New("invalid argument")
```
Comment (harshavardhana) [inline cmd/utils.go]: Just use 'errInvalidArgument'  or use `fmt.Errorf` to describe the error?

File: cmd/utils.go [hunk 2]
```diff
@@ -238,3 +241,25 @@ func dumpRequest(r *http.Request) string {
...
+	}
+	globalIsEnvBrowser = true
+	return strings.ToLower(os.Getenv("MINIO_BROWSER")), nil
```
Comment (harshavardhana) [inline cmd/utils.go]: why do "os.Getenv()" again? just use 'b'

File: cmd/utils.go [hunk 3]
```diff
@@ -238,3 +241,25 @@ func dumpRequest(r *http.Request) string {
 	}
 	return string(jsonBytes)
 }
+
+// Variant of getBrowserFromEnv but upon error fails right here.
+func mustGetBrowserFromEnv() string {
+	browser, err := getBrowserFromEnv()
+	if err != nil {
+		console.Fatalf("Unable to load browser parameter from environment. Err: %s.\n", err)
```
Comment (harshavardhana) [inline cmd/utils.go]: "Unable to load MINIO_BROWSER value from environment"

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Review by harshavardhana — COMMENTED** [review]
Can you also tests for these as well?

**Comment by vadmeste** [comment]
> Can you also tests for these as well?

Done

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**